validator/validation/validation.py

```diff
@@ -352,13 +366,26 @@ def run_validation(validation_id):
                                 validation_aborted = True
                                 __logger.debug('Validation got cancelled, dropping task {}: {}'.format(async_result.id, te))
 
-                if validation_aborted:
+                # in case there where no points with overlapping data within
+                # the validation period, results is an empty dictionary, and we
+                # count this job as error
+                if validation_aborted or not results:
                     validation_run.error_points += num_gpis_from_job(job_table[async_result.id])
                 else:
+                    results = pytesmo_to_qa4sm_results(results)
+
+                    # only count points where there was enough data as ok points
+                    # results is a dictionary with a single key
+                    # nobs = results[list(results.keys())[0]]["n_obs"]
+                    # ok_points = np.sum(nobs > 0)
+                    # error_points = len(nobs) - ok_points
+
                     check_and_store_results(async_result.id, results, run_dir)
+                    # validation_run.ok_points += ok_points
+                    # validation_run.error_points += error_points
                     validation_run.ok_points += num_gpis_from_job(job_table[async_result.id])
 
-            except Exception:
+            except Exception as e:
```

Comment:
This is not from your PR, but do you know what exception this refers to? Maybe we could narrow it down

Reply:
I had problems with validation runs that had no points, which was leading to problems (which is why I introduced the check for empty results in line 372).
I only added the `as e` for easier debugging.